### ISSUE TITLE
How to use distributed training on multiple GPUs

### ISSUE THREAD

**Issue by Zz-dong** [ContinualAI/avalanche#1477]
Can we now use distributed trainingon multiple gpu's.  I'm trying to use big models in avalanche, so distributed training is necessary and urgent for me. I've seen some issues working on this, but it doesn't seem to be finished.

**Comment by Zz-dong** [comment]
Is it possible for me to use [Accelerate](https://huggingface.co/docs/accelerate/index), Deepspeed, fp16 or gradient accumulation?

**Comment by AntonioCarta** [comment]
We don't have them integrated in Avalanche, but you can use some of Avalanche components in your own loops, for example [replay buffers](https://avalanche.continualai.org/how-tos/dataloading_buffers_replay). Other strategies can be used in a similar way, but you will have to adapt a bit the code.

For example, most regularization plugins have only two functions, `before_backward` and `after_training_experience`. If you call those in your training loop you can reuse them.
